packages/perseus/src/widgets/interactive-graphs/legacy-grid.tsx

```diff
@@ -0,0 +1,49 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import * as React from "react";
+
+import AssetContext from "../../asset-context";
+import {SvgImage} from "../../components";
+import {interactiveSizes} from "../../styles/constants";
+import {getInteractiveBoxFromSizeClass} from "../../util/sizing-utils";
+
+import type {PerseusImageBackground} from "../../perseus-types";
+import type {SizeClass} from "../../util/sizing-utils";
+
+/**
+ * If a graphie-to-png URL is provided in `backgroundImage`, will return the rendered graphie background.
```

Comment:
**terminology**: [graphie-to-png](https://khanacademy.atlassian.net/wiki/spaces/LC/pages/2535227849/Graph+Terminology#Graphie-to-png) is the name for the website to build Graphies.  I think in this case we can just call it a "graphie URL".

https://khanacademy.atlassian.net/wiki/spaces/LC/pages/2535227849/Graph+Terminology